### PR TITLE
Adds Toolkit Requirement Option

### PR DESCRIPTION
**Pull request by Bosenator**
Adds the option to require a toolkit, and the conditional statement to check for that presence.

I haven't been able to build and test this yet, as I'm on my laptop. It should work as is, and just wanted feed back before I get back and can test it/make any needed alterations.

### CODE REVIEW

File: addons/unflipping/functions/fn_hasToolKitRequired.sqf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
+ * Checks if the toolkit is required, and whether or not the unit has that item if it is
+ *
+ */
+params ["_unit"];
+
+if !(vet_unflipping_require_toolkit) exitWith { true }; // if it's not needed, return true for conditional sake -- this isn't a great methodology, I know
+if ("ToolKit" in (items _unit)) exitWith { true }; // has the toolkit, return true
+false;
```
Comment (veteran29) [inline addons/unflipping/functions/fn_hasToolKitRequired.sqf]: Last two lines could be simplified to:
`"ToolKit" in (items _unit)` (also no `;` after last statement please)

**Comment by veteran29** [comment]
Looks good, tested on my end and it works properly.

Thank you for your contribution!

**Comment by veteran29** [comment]
@Wyqer @corp-0 @topden, could you please take a look at new translation strings?

**Comment by Wyqer** [comment]
I’ll add the localizations this evening.

**Comment by corp-0** [comment]
sure thing

**Pull request merged**